openshift/mlflow-experiment-image-buildconfig.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: v1
```

Comment:
remove `-image-` from all the naming. A build config always produces an `image` :)